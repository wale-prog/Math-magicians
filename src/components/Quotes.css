.quote-container {
    line-height: 22px;
    padding: 3rem 0;
    border: 2px solid;
    margin: 0 auto 0 auto;
    width: 50%;
    text-align: center;
    border-radius: 15px;
    background-color: rgb(247, 241, 241);
}

.quote-container h3 {
    margin-bottom: 8px;
}