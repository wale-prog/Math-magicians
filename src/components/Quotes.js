/* eslint-disable */

import React from "react";
import './Quotes.css'

const Quotes = () => {
  return (
    <div className="quote-container">
      <p>
        {" "}
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus
        necessitatibus nostrum ullam aliquam, enim aspernatur
      </p>
    </div>
  );
};
export default Quotes;
