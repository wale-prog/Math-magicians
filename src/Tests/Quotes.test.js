import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

describe('Testing the Quote component', () => {
  it('Quotes is rendered', () => {
    const calc = renderer
      .create(<Quotes />)
      .toJSON();
    expect(calc).toMatchSnapshot();
  });
});
