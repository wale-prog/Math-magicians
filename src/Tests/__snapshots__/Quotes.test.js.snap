// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Quote component Quotes is rendered 1`] = `
<div
  className="quote-container"
>
  <h3>
    Quote of the day
  </h3>
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus necessitatibus nostrum ullam aliquam, enim aspernatur
  </p>
</div>
`;
